### PR TITLE
DCS-606 Bumping dependencies and testing k9s probe endpoint

### PR DESCRIPTION
**Pull request by andrewrlee**

### CODE REVIEW

File: dps-gradle-spring-boot-suppressions.xml [hunk 1]
```diff
@@ -54,4 +54,131 @@
     </packageUrl>
     <cve>CVE-2018-1258</cve>
   </suppress>
-</suppressions>
\ No newline at end of file
+
+  <suppress>
```
Comment (andymarke) [inline dps-gradle-spring-boot-suppressions.xml:58]: Shouldn't this be added to .ignore? This is the one generated by gradle at compile time right?

File: build.gradle.kts [hunk 2]
```diff
@@ -13,22 +13,22 @@ dependencies {
...
 
     implementation("org.springframework.boot:spring-boot-starter-data-jpa")
     implementation("org.springframework.boot:spring-boot-starter-cache")
     implementation("org.springframework.boot:spring-boot-starter-security")
     implementation("org.springframework.boot:spring-boot-starter-oauth2-resource-server")
     implementation("org.springframework.boot:spring-boot-starter-oauth2-client")
 
-    implementation("io.springfox:springfox-swagger2:2.9.2")
-    implementation("io.springfox:springfox-swagger-ui:2.9.2")
+    implementation("io.springfox:springfox-swagger2:3.0.0")
```
Comment (andymarke) [inline build.gradle.kts:25]: Some projects had issues with 3.0.0 I think

File: dps-gradle-spring-boot-suppressions.xml (hunk 1 above)
Comment (andrewrlee) [reply dps-gradle-spring-boot-suppressions.xml]: removed this

File: build.gradle.kts (hunk 2 above)
Comment (andrewrlee) [reply build.gradle.kts]: I've updated relevant config to get this to work. It generates random errors around void responses due to this bug : https://github.com/springfox/springfox/issues/3476 but it's only cosmetic.

**Pull request merged**